lib/confluence/vendor_specific_data.es6.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  name: 'VendorSpecificData',
+  package: 'org.chromium.apis.web',
+  documentation: `This class contains a browser and its
+    number of failure to ship APIs.`,
+  ids: ['browserName', 'date'],
+  properties: [
+    {
+      name: 'browserName',
+      class: 'String',
+      documentation: 'Name of the Browser',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'browser',
+      documentation: `An Browser object used to calculate failure to ship
+        metric with other verndors at this.date.`,
+      required: true,
+      final: true,
+    },
+    {
+      name: 'browserVersion',
+      class: 'String',
+      documentation: 'Version of this browser.',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'date',
+      class: 'Date',
+      documentation: 'The date at which this metric is computed.',
```

Comment:
nit: here and elsewhere: `for which` instead of `at which` makes clear that it is computed "for some historical date" not "at the (current) date".